feat(dashmate): add 'reindex' command (#533)
dashevo/platform: packages/dashmate/README.md

$ dashmate reset
```

### Reindex dashcore chain data

> 

Reviewer: ```suggestion

The `reindex` command rebuilds the block chain index using the downloaded block data.

```

> Sometimes you need to make a process of reindex for your core container, in case you imported new private keys, or your core data corrupted. The command `reindex` put your node in the reindex mode.

Reviewer: ```suggestion
Sometimes you need to run a reindex for your core container. This may be necessary if you import new private keys or if your core data has become corrupted. The `reindex` command puts your node into the reindex mode.
```

> It modifies the configuration and run the core container in `reindex=1` mode, wait until it fully resyncs, and brings it back in normal mode.

Reviewer: ```suggestion
It modifies the configuration, runs the core container in `reindex=1` mode, waits until it fully resynces, and returns it to the normal mode.
```

> The process is interactive (shows progress) and can be interrupted any time, but you cannot start your configuration until resync is fully complete.

Reviewer: ```suggestion
The process is interactive (shows progress) and can be interrupted any time, but you cannot start your configuration until the resync is fully complete.
```

> Command is working both for regular and local configurations

Reviewer: ```suggestion
The `reindex` command works for regular and local configurations.

```

$ dashmate reset
```

### Reindex dashcore chain data

The `reindex` command rebuilds the block chain index using the downloaded block data.

It modifies the configuration, runs the core container in `reindex=1` mode, waits until it fully resynces, and returns it to the normal mode.

The process is interactive (shows progress) and can be interrupted any time, but you cannot start your configuration until the resync is fully complete.

The `reindex` command works for regular and local configurations.

```

> Reindex core container

Reviewer 2: Update help info
